client/components/notice/style.scss

```diff
@@ -41,6 +41,10 @@
 	}
 }
 
+.notice__icon-wrapper {
```

Comment:
@breezyskies @mikeshelton1503 I had to make a CSS hack to prevent notice icons getting smaller as the number of lines are getting higher. Is there proper solution to this? I think this is a regression.